src/components/Timer/TimerStartDialog.tsx

```
@@ -0,0 +1,171 @@
+                    className="h-14"
+                  />
+                  <span
+                    onClick={handleAddTask}
+                    className={cn(
+                      "typography-body-small-m absolute right-6 cursor-pointer",
+                      !todayGoal.trim() ? "text-gray-400" : "text-primary-0",
+                    )}
+                  >
+                    추가
+                  </span>
+                </div>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)